pages/2020/schedule.rst

```diff
@@ -1,258 +1,66 @@
-  |12:30pm |Lunch                                                                           |
-  +--------+                                                                                |
-  |1:00pm  |                                                                                |
-  +--------+--------------------------------------------------------------------------------+
-  |1:30pm  |Unconferences                                                                   |
-  +--------+                                                                                |
-  |2:00pm  |                                                                                |
-  +--------+--------------------------------------------------------------------------------+
-  |2:30pm  |Unconferences                                                                   |
-  +--------+                                                                                |
-  |3:00pm  |                                                                                |
-  +--------+--------------------------------------------------------------------------------+
-  |3:30pm  |Coffee                                                                          |
-  +--------+--------------------------------------------------------------------------------+
-  |4:00pm  |Lightning talks                                                                 |
-  +--------+--------------------------------------------------------------------------------+
-  |4:30pm  |Hack-day pre-pitches                                                            |
-  +--------+--------------------------------------------------------------------------------+
-  |5:00pm  |Summary of unconferences                                                        |
-  +--------+--------------------------------------------------------------------------------+
 
 :raw-html:`<br />`
 
-
 .. table::
   :widths: 10, 100
+  :align: center
+
+
+  +------------+--------------------------------------------------------------------------------+
+  | Time (UTC) | Thursday (23 April)                                                            |
+  +============+================================================================================+
+  | 0:00       | Hacking continues                                                              |
+  +------------+--------------------------------------------------------------------------------+
+  | 7:50       | *Sophie Murray* - **Opening Remarks**                                          |
+  +------------+--------------------------------------------------------------------------------+
+  | 8:00       | *Brett Morris* - **State of the Package: Astropy**                             |
+  +------------+--------------------------------------------------------------------------------+
+  | 9:00       | *Speaker* - **Talk Title**                                                     |
```

Comment:
Possibly replace this with something more official-sounding perhaps, "Talk TBD" or something? This seems a little too obviously placeholdery :P